tests/loadtests/glloadtests/shader-based/shaders.cpp

```diff
@@ -2,38 +2,16 @@
 /* vi: set sw=2 ts=4 expandtab: */
 
 /*
- * ©2008 HI Corporation.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright 2008 HI Corporation.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
 /**
  * @file    shaders.c
  * @brief   Shaders used by the DrawTexture and TexturedCube samples.
  *
- * @author Mark Callow, www.edgewise-consulting.com
- */
-
-#include "GL/glcorearb.h"
-#include "ktx.h"
-
-const GLchar* pszVs =
-"layout(location = 0) in vec4 position;\n"
-"layout(location = 1) in vec4 color;\n"
-"layout(location = 2) in vec4 texcoord;\n\n"
-
-"out vec4 v_color;\n"
-"out vec2 v_texcoord;\n\n"
+ * @author-2020 Mark Callow, www.edgewise-consulting.com
+ */ SPDX-License-Identifier: Apache-2.0
```

Comment:
Delete these 2 lines? They look like  mistake. If they're meant to be here, an opening "/*" is needed.